src/api/tests/service/billing/test_credit_notifications.py

```
@@ -0,0 +1,503 @@
+    app = Flask(__name__)
+    app.testing = True
+    app.config.update(
+        SQLALCHEMY_DATABASE_URI=db_uri,
+        SQLALCHEMY_BINDS={
+            "ai_shifu_saas": db_uri,
+            "ai_shifu_admin": db_uri,
+        },
+        SQLALCHEMY_TRACK_MODIFICATIONS=False,
+        SQLALCHEMY_ENGINE_OPTIONS={"connect_args": {"check_same_thread": False}},
+        REDIS_KEY_PREFIX="credit-notification-test:",
+        SECRET_KEY="credit-notification-secret",
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "SECRET_KEY"

(S106)